src/Raven.Server/ServerWide/Commands/AddDatabaseCommand.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.ServerWide.Commands
+{
+    public class AddDatabaseCommand : CommandBase
+    {
+        public string Name;
+        public BlittableJsonReaderObject Record;
+        public bool Encrypted;
+        public override DynamicJsonValue ToJson()
+        {
+            //this is just for validating that this is a valid database record
+            JsonDeserializationCluster.DatabaseRecord(Record);
+            return new DynamicJsonValue
+            {
+                ["Type"] = nameof(AddDatabaseCommand),
```

Comment:
Maybe CommandBase can add Type, so we would not have to 'remember' about it?
e.g.
var json = base.ToJson();
//fill here with data
return json;